### PR TITLE
Fixes #3178 NTR placental terms

### LINKED ISSUE #3178: NTR placental terms
**Preferred term label:**
placental disc

**Synonyms**
placental disk

**Definition (free text, please give PubMed ID)**
A disc-shaped structure that is part of the placenta. A placental disc supports fetal development through the exchange of nutrients between maternal and fetal bloodstreams. The placental disc is structured to keep the blood streams separate, while maximizing surface area for the transport of nutrients between them, to maintain an immunological barrier.
[PMID:35496035](https://pubmed.ncbi.nlm.nih.gov/35496035)

**Parent term (use https://www.ebi.ac.uk/ols/ontologies/uberon)**
'part of' some placenta
'anatomical structure'

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-6677-8489

*'intervillous space', 'chorionic plate', 'basal plate' and 'chorionic tree' need to add `'part of' some 'placental disc'`.*

--------

**Preferred term label:**
chorionic villous mesenchyme

**Synonyms**

**Definition (free text, please give PubMed ID)**
A mesenchyme that is part of a chorionic villus.
[doi:10.1007/978-3-030-84725-8](https://doi.org/10.1007/978-3-030-84725-8)

**Parent term (use https://www.ebi.ac.uk/ols/ontologies/uberon)**
mesenchyme and ('part of' some 'chorionic villus')

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-6677-8489

--------

**Preferred term label:**
chorionic stem villous blood vessel

**Synonyms**
stem villous vessel BROAD PMID:7547738

**Definition (free text, please give PubMed ID)**
A blood vessel that is part of a chorionic stem villus.
[doi:10.1007/978-3-030-84725-8](https://doi.org/10.1007/978-3-030-84725-8)

**Parent term (use https://www.ebi.ac.uk/ols/ontologies/uberon)**
'blood vessel' and ('part of' some 'chorionic stem villus')

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-6677-8489

--------

**Preferred term label:**
chorionic terminal villous capillary

**Synonyms**
terminal villous capillary BROAD PMID:22461195

**Definition (free text, please give PubMed ID)**
A capillary that is part of a chorionic terminal villus. This capillary is generally narrow but display sinusoidal dilations along its length, often at the points of acute bends. These dilations bring the capillary endothelium into close contact with the overlying syncytiotrophoblast to form a vasculosyncytial membrane. 
[doi:10.1007/978-3-030-84725-8](https://doi.org/10.1007/978-3-030-84725-8)

**Parent term (use https://www.ebi.ac.uk/ols/ontologies/uberon)**
capillary and ('part of' some 'chorionic terminal villus')

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-6677-8489

--------

**Preferred term label:**
endometrial lymphatic vessel

**Synonyms**

**Definition (free text, please give PubMed ID)**
A lymphatic vessel that is part of an endometrium.
[PMID:23054374](https://link.springer.com/article/10.1007/s11154-012-9224-6)

**Parent term (use https://www.ebi.ac.uk/ols/ontologies/uberon)**
'lymphatic vessel' and ('part of' some endometrium)

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-6677-8489

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #3178 NTR placental terms

### CODE REVIEW

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Review by zoependlington — APPROVED** [review]

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224633,6 +224634,54 @@ relationship: part_of UBERON:0001294 ! inner medulla of kidney
...
+def: "A disc-shaped structure that is part of the placenta. A placental disc supports fetal development through the exchange of nutrients between maternal and fetal bloodstreams. The placental disc is structured to keep the blood streams separate, while maximizing surface area for the transport of nutrients between them, to maintain an immunological barrier." [PMID:35496035]
+synonym: "placental disk" EXACT [PMID:30133439]
+is_a: UBERON:0000061 ! anatomical structure
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-Barbé
+relationship: part_of UBERON:0001987 ! placenta
+property_value: dcterms-date "2024-01-16T17:50:20Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600039
+name: chorionic villous mesenchyme
```
Comment (cmungall) [inline src/ontology/uberon-edit.obo:224649]: this term is great but I would expect it to be an intermediate in the develops_from graph, e.g chorionic vasculature develops_from CVM...?

Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224649]: Thank you for noticing, I will add a ticket.

**Pull request merged**